ehsm_kms_service/function.js

```diff
@@ -333,6 +342,26 @@ const _checkPayload = function (req, res) {
           return false
         }
       }
+      if (currentPayLoad[key].type == 'base64' && payload[key]) {
+        if (typeof payload[key] != 'string') {
+          res.send(_result(400, `${key} must be of base64 string type`))
+          return false
+        }
+        if (
+          payload[key] != undefined &&
+          ((currentPayLoad[key].maxLength &&
+            payload[key].length > currentPayLoad[key].maxLength) ||
+            (currentPayLoad[key].minLength &&
+              payload[key].length < currentPayLoad[key].minLength))
+        ) {
+          res.send(_result(400, `${key} length error`))
+          return false
+        }

```

Comment:
Can we special handle aad='' here like:

if (!currentPayLoad[key].minLength && payload[key].length === 0)
return true

not sure, please confirm.

Reply:
This case has been covered at line 322 - 328